examples/step-1/doc/intro.dox

```diff
@@ -1,4 +1,4 @@
-<a name="Intro"></a>
+<a name="step-1-Intro"></a>
```

Comment:
For all of these anchors in the tutorial programs, have you searched whether there are references to these names? As in `<a href="..."`?

Reply:
For the intro and result files this is done in `intro2toc.pl` and I added a prefix parameter to it as well.